### PR TITLE
CORDA-1938 Adding version info to the node info submission request

### PR DESCRIPTION
**Pull request by mkit**
This PR adds version info to the node info submission request.

### CODE REVIEW

File: node/src/main/kotlin/net/corda/node/services/network/NetworkMapClient.kt [hunk 1]
```diff
@@ -45,7 +46,9 @@ class NetworkMapClient(compatibilityZoneURL: URL) {
     fun ackNetworkParametersUpdate(signedParametersHash: SignedData<SecureHash>) {
         val ackURL = URL("$networkMapUrl/ack-parameters")
         logger.trace { "Sending network parameters with hash ${signedParametersHash.raw.deserialize()} approval to $ackURL." }
-        ackURL.post(signedParametersHash.serialize())
+        ackURL.post(signedParametersHash.serialize(),
+                "Platform-Version" to "${versionInfo.platformVersion}",
```
Comment (kasiastreich) [inline node/src/main/kotlin/net/corda/node/services/network/NetworkMapClient.kt]: Platform version is stored in `NodeInfo` already

File: node/src/main/kotlin/net/corda/node/services/network/NetworkMapClient.kt [hunk 2]
```diff
@@ -45,7 +46,9 @@ class NetworkMapClient(compatibilityZoneURL: URL) {
...
+                "Client-Version" to versionInfo.releaseVersion)
```
Comment (kasiastreich) [inline node/src/main/kotlin/net/corda/node/services/network/NetworkMapClient.kt]: Also, do we need that much granularity?

File: node/src/main/kotlin/net/corda/node/services/network/NetworkMapClient.kt (hunk 1 above)
Comment (mkit) [reply node/src/main/kotlin/net/corda/node/services/network/NetworkMapClient.kt]: Yup but for the consistency with the CSR submission we unify the headers.

File: node/src/main/kotlin/net/corda/node/services/network/NetworkMapClient.kt (hunk 2 above)
Comment (mkit) [reply node/src/main/kotlin/net/corda/node/services/network/NetworkMapClient.kt]: We might in the future.

Comment (fenryka) [reply node/src/main/kotlin/net/corda/node/services/network/NetworkMapClient.kt]: We do for the cert version change, plus it's potentially useful

File: node/src/main/kotlin/net/corda/node/services/network/NetworkMapClient.kt (hunk 1 above)
Comment (fenryka) [reply node/src/main/kotlin/net/corda/node/services/network/NetworkMapClient.kt]: useful to have in the header though so we don't have to pull apart the blob before deciding what to do

**Pull request merged**